src/data-visualisation/decluttering/decluttering.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: collection
+title: "Decluttering and pulling focus"
+description: "Guidance on the importance of decluttering and pulling focus for data visualisation"
+status: DRAFT
+tags: data
+order: 1
+collection_tag: dv-declutter
+pagination:
+  data: collections.dv-declutter
+  size: 50
+  alias: articles
+---
+## Importance of decluttering  
+  
+To ensure users can quickly make effective and informed decisions, it is important to give the user all information necessary, without providing too much to distract the user.  
```

Comment:
Has a data vis peer reviewed the content?

Reply:
Helen wrote the content, I copied up and made slight edits, Helen re-reviewed